specification/eventhub/resource-manager/Microsoft.EventHub/2017-04-01/EventHub.json

```diff
@@ -1904,6 +2153,68 @@
           "type": "string"
         }
       }
+    },
+    "ArmDisasterRecovery": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "properties": {
+            "provisioningState": {
```

Comment:
enum. What about a Failed provisioning State value? You mention that possible values as accepted or succeeded. In case the operation fails, failed should be returned. And do you support cancel? In that case, you will have a canceled provisioning state as well.

Reply:
Resolved, we don't support cancel.